app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -0,0 +1,1491 @@
+// Foundation by ZURB
```

Comment:
Throw this file in your `vendor/assets/stylesheets` folder so your codeclimate score doesn't plummet